modules/apps/data-engine/data-engine-js-components-web/src/main/resources/META-INF/resources/js/custom/form-report/index.scss

```diff
@@ -498,29 +565,32 @@
 		min-height: 500px;
 	}
 
-	.report-cards-area {
+	&--cards-area {
+		margin-left: 20px;
 		transition: all 0.3s cubic-bezier(0.25, 0.11, 0.25, 1);
-		width: 806px;
+		width: 80%;
+		width: 808px;
```

Comment:
I think here you only meant to use the `width: 808px;` and not the 80% one, could you verify please?